x-pack/metricbeat/module/sql/_meta/docs.asciidoc

```diff
@@ -469,3 +469,61 @@ The example generates this event: By disabling the flag `raw_data.enabled`, whic
   }
 }
 ----
+
+[float]
+=== Example: Merge multiple queries to single event.
+
+Multiple queries will create multiple events, one for each query.  It may be preferrable to create a single event by combining the metrics together in a single event.
+
+This feature can be enabled using the `request_merge` config.
+
+However, such a merge is possible only if the table queries being merged, each produce a single row.
+
+For example:
+
+[source,yaml]
+----
+- module: sql
+  metricsets:
+    - query
+  period: 10s
+  hosts: ["postgres://postgres:postgres@localhost?sslmode=disable"]
+  driver: "postgres"
+  raw_data.enabled: true
+  request_merge: true
+  sql_queries:
+    - query: "SELECT blks_hit FROM pg_stat_database limit 1;"
+      response_format: table
+    - query: "SELECT blks_read FROM pg_stat_database limit 1;"
+      response_format: table
```

Comment:
How would this configuration be different to the following one?
```suggestion
  sql_queries:
    - query: "SELECT blks_hit, blks_read FROM pg_stat_database limit 1;"
      response_format: table
```

If this would generate the same result, please put here an example of a case where `request_merge` would be the only option. I guess that this merge is required in cases where data from multiple tables is being collected.

Reply:
Updated the example with example queries from two different tables.